app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/viewmodels/DataCollectionViewModel.kt

```diff
@@ -51,8 +54,8 @@ class DataCollectionViewModel(application: Application, val mapViewModel: MapVie
     private lateinit var portal: Portal
     private lateinit var portalItem: PortalItem
 
-    private val _isDrawerClosed: MutableLiveData<Boolean> = MutableLiveData(false)
-    val isDrawerClosed: LiveData<Boolean> = _isDrawerClosed
+    private val _closeDrawerEvent = MutableLiveData<Event<Unit>>()
+    val closeDrawerEvent: LiveData<Event<Unit>> = _closeDrawerEvent
```

Comment:
Why do you prefer `Event` over `SingleLiveEvent`?

Reply:
I am not sure of the question here.

If I don't have the DrawerClosing as an `Event` then the open drawer will be closed on device configuration change..

Reply:
Looks like `SingleLiveEvent` is just a different approach to achieve the same thing.